weblate_web/templates/hosting.html

```diff
@@ -657,8 +657,8 @@ <h2 class="section-title crypto payment-conditions" id="strings">{% trans "Sourc
             <div class="libre-hosting" id="libre">
               <h2>{% trans "Weblate proudly supports libre projects" %}</h2>
               <p>
-                {% blocktrans with url="https://hosted.weblate.org/hosting/" %}If you feel like being supported by Weblate will help you, <a href="{{ url }}">set up your libre project</a> and get the <strong>Libre plan</strong> gratis.{% endblocktrans %}
-                {% blocktrans %}It has the same limits as the <strong>Advanced plan</strong>, but only for public projects.{% endblocktrans %}
+                {% blocktrans with url="https://hosted.weblate.org/hosting/" %}<a href="{{ url }}">Set up your libre project</a> using the <strong>Libre plan</strong> gratis.{% endblocktrans %}
+                {% blocktrans %}The same as the <strong>Advanced plan</strong>, for your libre software project.{% endblocktrans %}
```

Comment:
The intention here was to write that the translated project has to be public on Hosted Weblate, private/protected access choices are unavailable in the Libre plan.

Reply:
Didn't see this comment because I stopped paying attention to all the updates from the bot.
The thing to avoid in

[It has the same limits as the <strong>160k plan</strong>, but only for public projects](https://github.com/WeblateOrg/website/pull/1937/files#diff-45991c2ff8708c73daab5be090374d6367a8a947c4b4d2e94d3cf5de88d643e5L747)

is thinking that only the limits apply to public projects, and _not_ private ones.
I get that it is possible to reasonably deduct this isn't the case, but it is hard to read for no reason.

The general problem and what is continued through

[It has the same limits as the <strong>160k plan</strong>, `and is` only for public projects.](https://github.com/WeblateOrg/website/pull/1937/files#diff-45991c2ff8708c73daab5be090374d6367a8a947c4b4d2e94d3cf5de88d643e5R747)

is something that makes a point for people that somehow have libre projects, that are non-public.

Yes that can happen, but does it ever?

In the event that it does happen, and specifically for only the users attempting to do so, I think it is perfectly reasonable to explain this isn't how the libre plan works. That is a type of deduction that works to the benefit of what info is left behind, because it is a corollary.

Moreover, the libre plan actually _is_ explained at a later stage, namely in https://hosted.weblate.org/hosting/
which is the link in said text in https://weblate.org/en/hosting/

---

If the addition of the clause somehow makes sense to put in, I can of course do so in maybe a different sentence.

Reply:
I understand, it is easy to get lost in all the notifications.

If you have a suggestion(s) to make that sentence better, please post it here. I will think about it and try to figure out something as well.

It doesn't seem to be much of an issue, there is a continuous and rising stream of libre projects, but if we can make the description better, let's do it. We want to be more clear and accessible.

Reply:
It is a giant issue. Every time a libre software project ends up using a non-libre platform for what is no good reason that project is now harder to work on. Its translators are subjected to garbage, and the terms for the accruing translation memory are worse.

It is a lot of effort for me to direct libre software projects to Weblate.
That text doesn't help.
The amount of projects already using Weblate does help.

That situation needs improvement.
We now have non-english, and still known to be non-functional directions.
That is not a good luck for a project dealing in language…

Where does your confidence come from, and how do you skip the part of my comment detailing my view?

You are defending something I can't understand. How am I meant to cook up a sentence for a concern I don't understand. That is precisely where you just failed.

My concerns are valid and in writing. Your concern is who knows and only in your head.